packages/ketcher-macromolecules/src/components/SequenceSyncEditModeButton/SequenceSyncEditModeButton.tsx

```diff
@@ -0,0 +1,164 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import { useEffect, useState } from 'react';
+import { useAppSelector, useLayoutMode } from 'hooks';
+import { hasAntisenseChains, selectEditor } from 'state/common';
+import { ModeTypes } from 'ketcher-core';
+import styled from '@emotion/styled';
+import { Button } from 'ketcher-react';
+import { blurActiveElement } from 'helpers/canvas';
+
+const StyledButton = styled(Button)<{ isActive?: boolean }>(
+  ({ theme, isActive }) => ({
+    width: '40px',
+    height: '31px',
+    backgroundColor: isActive
+      ? theme.ketcher.color.button.group.active
+      : 'white',
+    marginRight: '8px',
+    border: isActive
+      ? theme.ketcher.outline.selected.color
+      : theme.ketcher.outline.small,
+    borderRadius: '4px',
+    outline: 'none',
+
+    ':hover': {
+      backgroundColor: isActive
+        ? theme.ketcher.color.button.group.hover
+        : 'white',
+    },
+
+    ':hover svg': {
+      fill: isActive ? 'white' : theme.ketcher.color.button.group.active,
+    },
+  }),
+);
+
+export const SequenceSyncEditModeButton = () => {
+  const editor = useAppSelector(selectEditor);
+  const [isSequenceSyncEditMode, setIsSequenceSyncEditMode] = useState(true);
```

Comment:
I suppose we don't need to have this state here, it's not a responsibility of the button to track the sync mode state, it should just rely on the editor's internal state on render stage